<launch>
    <!-- PARAMS -->
    <param name="rgb_image_topic" value="/camera/rgb/image_raw" />
    <!-- <param name="rgb_image_topic" value="/camera/rgb/image_rect_color" /> -->
    <param name="depth_image_topic" value="/camera/depth_registered/image_raw" />
    <param name="camera_info_topic" value="/camera/rgb/camera_info" />
    <param name="camera_fps" value="30" />
    <param name="wanted_fps" value="5" />

    <param name="rgb_save_path" value="/home/RVLuser/data/summit_test/rgb" />
    <param name="rgb_seg_save_path" value="/home/RVLuser/data/summit_test/rgb_seg" />
    <param name="depth_save_path" value="/home/RVLuser/data/summit_test/depth" />
    <param name="depth_seg_save_path" value="/home/RVLuser/data/summit_test/depth_seg" />
    <param name="ply_save_path" value="/home/RVLuser/data/summit_test/ply_seg" />
    <param name="rvl_cfg_path"
        value="/home/RVLuser/rvl-linux/RVLRecognitionDemo_Cupec_DDD2_Detection.cfg" />

    <param name="bag_path" value="/home/RVLuser/data/bags/summit_test_last.bag" />

    <!-- NODES -->

    <param name="load_model" type="bool" value="true" />

    <node name="node_wait" pkg="ao_manipulation" type="node_wait.py" output="screen"/>


    <node name="only_save_images_node" pkg="ao_manipulation" type="node_save_images_plys2.py" output="screen" >
        <param name="bag_used" type="bool" value="false" />
        <param name="skip" type="bool" value="false" />
    </node>

    <!-- <node name="only_detect_cabinet_model_node" pkg="ao_manipulation" type="node_only_detect_cabinet.py"
        output="screen" /> -->

</launch>